tests/e2e/scale/test_scale_3_OCS_worker_nodes_and_1500_PVCs.py

```diff
@@ -111,6 +111,70 @@ def test_pv_scale_out_create_pvcs_and_respin_ceph_pods(
         )
 
 
+@scale
+@ignore_leftovers
+@skipif_external_mode
```

Comment:
this test can run even on external mode, any reason for skipping it on it?

Reply:
We haven't tested any of the scale tests in external mode, it shouldn't be problem to remove this, anyway let me know WDYT.

Reply:
ah ok. we can allow the test to run once you start scale tests on external mode.